src/pages/Items/InquiryItem.jsx

```diff
@@ -0,0 +1,142 @@
+import styled from "@emotion/styled";
+import { useState } from "react";
+
+const ItemWrapper = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+
+  border-bottom: 1px solid #e5e7eb;
+  padding-bottom: 12px;
+  background-color: #ffffff;
+
+  font-family: Pretendard;
+  font-size: 14px;
+  line-height: 24px;
+  color: #1f2937;
+`;
+
+const QuestionText = styled.div``;
+
+const EditInput = styled.input`
```

Comment:
❗️ 수정요청
여러줄 입력이 가능한 textarea가 더 적절할 것 같아요!

```suggestion
const EditInput = styled.textarea`
```